_includes/v22.2/ui/insights.md

```diff
@@ -0,0 +1,224 @@
+## Workload Insights tab
+
+The **Workload Insights** tab displays insights related to transaction and statement executions.
+
+### Transaction Executions view
+
+{% if page.cloud != true -%}
+To display this view, click **Insights** in the left-hand navigation of the DB Console and select **Workload Insights > Transaction Executions**.
+{% endif -%}
+{% if page.cloud == true -%}
+{{site.data.alerts.callout_info}}
+The **Transaction Executions** view is currently unavailable for {{ site.data.products.serverless }} clusters.
+{{site.data.alerts.end}}
+
+To display this view, click **Insights** in the left-hand navigation of the Cloud Console and select **Workload Insights > Transaction Executions**.
+{% endif -%}
+
+The **Transaction Executions** view provides an overview of all transaction executions that have been flagged with insights.
+
+{{site.data.alerts.callout_info}}
+The rows in this page are populated from the [`crdb_internal.transaction_contention_events`]({{ link_prefix }}crdb-internal.html#transaction_contention_events) table.
+
+- The results displayed in the **Transaction Executions** view will be available as long as the corresponding row in the [`crdb_internal.transaction_contention_events`]({{ link_prefix }}crdb-internal.html#transaction_contention_events) table exists and as long as the rows in each node use less space than `sql.contention.event_store.capacity`.
```

Comment:
I don't mind having documenting this detail in case users want to programmatically pull information via SQL.

To clarify,  `sql.contention.event_store.capacity` controls whether there are rows in crdb_internal.transaction_contention_events. If the capacity is reached, the oldest row gets evicted.

So we can explain the relationship a bit better and avoid 'and as long as':

- The results displayed in the **Transaction Executions** view will be available as long as the corresponding row in the [`crdb_internal.transaction_contention_events`]({{ link_prefix }}crdb-internal.html#transaction_contention_events) table exists. Rows will exist as long as `sql.contention.event_store.capacity` is not reached in each node.